setup.cfg

```diff
@@ -13,7 +13,7 @@ addopts =
     --ignore maint_tools
     --doctest-modules
     --disable-pytest-warnings
-    -rs
+    -rxXs
```

Comment:
This says to show XFAIL and XPASS tests in the final summary report (that previously only included SKIP)